### DISCUSSION TITLE
扫描c段

### DISCUSSION THREAD

**Discussion by jinzezhi** [projectdiscovery/httpx#1315] [General]
扫描c段时一直卡着不动 等了将近半小时没有输出1个结果 是我命令不对吗
![image](https://user-images.githubusercontent.com/93123422/231100914-e24caca2-5a69-4976-81d2-b619b61ae3ec.png)

**Comment by Mzack9999** [comment]
Most likely the the long delay is due to the default timeout settings with retries that hits all ports. I'd recommend to perform a SYN portscan prior to use httpx with such big ranges.
